docs/cloud/unity-delta-sharing.md

```diff
@@ -7,37 +7,30 @@ has_children: false
 ---
 
 # Unity Delta Sharing
+{: .d-inline-block }
+lakeFS Cloud
+{: .label .label-green }
 
 ## Introduction
 
-lakeFS Unity Delta Sharing provides a read-only experience from Unity
-Catalog for lakeFS customers.  Currently this is available as a private
-preview.  It provides _full read-only functionality_ for Unity Catalog.  It
-does _not_ provide a "self-service" experience to set up the service.
+lakeFS Unity Delta Sharing provides a read-only experience from Unity Catalog for lakeFS customers.  Currently, this is available as a private
+preview.  It provides _full read-only functionality_ for Unity Catalog.  It does _not_ provide a "self-service" experience to set up the service.
 
 ## Setup
 
-This guide explains how to set up and use lakeFS Delta Sharing.  Currently
-you will have to configure lakeFS Delta Sharing in collaboration with
-Treeverse Customer Success.  Once set up is complete, you will of course be
-able to use lakeFS Delta Sharing on existing and on new tables without
+This guide explains how to set up and use lakeFS Delta Sharing.  Currently, you will have to configure lakeFS Delta Sharing in collaboration with
+Treeverse Customer Success.  Once set up is complete, you will of course be able to use lakeFS Delta Sharing on existing and on new tables without
 further assistance.
 
 ### 1. Collect data and initial setup
 
-* Select a "Delta Sharing configuration URL".  This is a single location on lakeFS
-  to hold the top-level configuration of lakeFS Delta Sharing across all
-  repositories of your organization.  Typically it will have the form
-  `lakefs://REPO/main/lakefs_delta_sharing.yaml` for one of your
-  repositories.  A longer path may be obviously be supplied.  However we do
-  recommend keeping it on the `main` branch, as this object represents state
+* Select a `Delta Sharing configuration URL`.  This is a single location on lakeFS to hold the top-level configuration of lakeFS Delta Sharing across all
+  repositories of your organization.  Typically, it will have the form `lakefs://REPO/main/lakefs_delta_sharing.yaml` for one of your
+  repositories.  A longer path may be supplied - however, we do recommend keeping it on the `main` branch, as this object represents state
   for the entire installation.
 
-* Create a user `lakefs-delta-sharing-service` for lakeFS Delta Sharing, and
-  an access key for that user.  It should have at least read permissions for
-  the configuration URL and for all repositories and all data accesses by
-  Unity.  lakeFS Delta Sharing will these credentials to communicate with
-  lakeFS.
+* Create a user `lakefs-delta-sharing-service` for lakeFS Delta Sharing, and an access key for that user.  It should have at least read permissions for
+  the configuration URL and for all repositories and all data accesses by Unity.  lakeFS Delta Sharing will these credentials to communicate with lakeFS.
```

Comment:
it's harder to review here since the line wrap width has changed. Is there any net difference in the text?

Reply:
No change in text